recipe/meta.yaml

```diff
@@ -55,10 +55,12 @@ outputs:
         - pytorch {{ pytorch }} {{ torch_proc_type }}*
```

Comment:
probably want the same in host?
```suggestion
        - pytorch
```

Reply:
Um, i guess we should add the torch_proc_type in the host too. the thing is, we *need* the GPU stuff (as far as I know).

Reply:
I added a comment in the yaml as to why we can have two constrains in the building sections, but not in the running sections.

Reply:
Fine by me. 👍